CMakeLists.txt

```diff
@@ -365,7 +365,7 @@ if(ENABLE_COVERAGE)
     DEPENDS check)
 
   add_custom_target(coverage
-    COMMAND ${GENHTML_PATH} --prefix ${CMAKE_CURRENT_BINARY_DIR} --output-directory coverage --title "Test Coverage" --legend --show-details coverage.info
+    COMMAND ${GENHTML_PATH} --prefix ${CMAKE_CURRENT_BINARY_DIR} --output-directory coverage --title "Test Coverage" --legend --show-details coverage.info --ignore-errors=negative
```

Comment:
genhtml was running into an error where the line hit count was negative at some point. Documentation says it's possibly related to [a GCC bug](https://manpages.debian.org/unstable/lcov/genhtml.1.en.html#negative:).